Misc/NEWS.d/next/Library/2020-07-19-20-10-41.bpo-41229.p8rJa2.rst

```diff
@@ -0,0 +1,2 @@
+Add ``contextlib.aclosing`` analogous to ``contextlib.closing`` but for
+async generators and arbitrary objects with an ``aclose`` coroutine method.
```

Comment:
it would be useful to mention the primary motivation (deterministic cleanup of async generators)

Reply:
Updated!